### PR TITLE
[contributing] add syncing up with the upstream

### PR DESCRIPTION
**Pull request by ydcjeff**
Fixes #{issue number} 

Description:

Add how to sync up with the upstream in CONTRIBUTING.md

Check list:
* [ ] New tests are added (if a new feature is added)
* [ ] New doc strings: description and/or example code are in RST format
* [ ] Documentation is updated (if required)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -167,6 +167,40 @@ If you are not familiar with creating a Pull Request, here are some guides:
...
+
+Now you can get the latest development into your forked repository with this:
+
+```bash
+git fetch upstream
+git checkout master
+git merge upstream/master
+```
+
+Rebasing for a PR
```
Comment (vfdev-5) [inline CONTRIBUTING.md:195]: We do not require rebasing on master, actually. As all commit history in the PR is squashed into a single commit. Maybe, we can simply mention that rebasing or merging, everything is possible as we finally squash.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -167,6 +167,40 @@ If you are not familiar with creating a Pull Request, here are some guides:
 - http://stackoverflow.com/questions/14680711/how-to-do-a-github-pull-request
 - https://help.github.com/articles/creating-a-pull-request/
 
+#### Sync up with the upstream
```
Comment (vfdev-5) [inline CONTRIBUTING.md:170]: I'd say this is a sort of subsection of "Send a PR" right ?
```suggestion
##### Sync up with the upstream
```

**Review by vfdev-5 — COMMENTED** [review]
Thanks for the PR @ydcjeff !
I left few nit comments to update it.

File: CONTRIBUTING.md (hunk 1 above)
Comment (ydcjeff) [reply CONTRIBUTING.md:195]: I was writing that initially for conflicting PR
I know we squashed and merged into master.
But not sure how you want to suggest for a conflicting PR?

File: CONTRIBUTING.md (hunk 2 above)
Comment (ydcjeff) [reply CONTRIBUTING.md:170]: At first, I thought this for updating the forked repo with upstream.
but yes, it makes more sense under "Send a PR"

File: CONTRIBUTING.md (hunk 1 above)
Comment (vfdev-5) [reply CONTRIBUTING.md:195]: > But not sure how you want to suggest for a conflicting PR?

Good question. I'd say that we can suggest to use "merge" and resolve the conflict once.

Comment (ydcjeff) [reply CONTRIBUTING.md:195]: Alright!

Comment (vfdev-5) [reply CONTRIBUTING.md:195]: Maybe we can remove the paragraph about rebasing, right ?

Comment (ydcjeff) [reply CONTRIBUTING.md:195]: Yes if you wish

Comment (ydcjeff) [reply CONTRIBUTING.md]: Done!

**Review by vfdev-5 — APPROVED** [review]
Thanks @ydcjeff, LGTM !

**Pull request merged**